File: poc-challenges/250731/src/test/java/com/tdd/domain/FactorialTest.java
package com.tdd.domain;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

public class FactorialTest {
    @Test
    void factorial_test() {
        assertEquals(0, fac(0));
    }

    private int fac(int n) {
        return 0;
    }
}

File: poc-challenges/250906/src/test/kotlin/com/currenjin/tdd/FactorialTest.kt
package com.currenjin.tdd

import org.junit.jupiter.api.Test
import kotlin.test.assertEquals

class FactorialTest {
    @Test
    fun factorial() {
        assertEquals(1, Factorial.fac(1))
        assertEquals(2, Factorial.fac(2))
        assertEquals(6, Factorial.fac(3))
    }
}